src/sentry/static/sentry/app/views/organizationIntegrations/docIntegrationDetailedView.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import Button from 'app/components/button';
+import space from 'app/styles/space';
+import {t} from 'app/locale';
+import {DocumentIntegration} from 'app/types';
+import withOrganization from 'app/utils/withOrganization';
+
+import AbstractIntegrationDetailedView from './abstractIntegrationDetailedView';
+import {documentIntegrations} from './constants';
+
+type Tab = AbstractIntegrationDetailedView['state']['tab'];
+
+class SentryAppDetailedView extends AbstractIntegrationDetailedView<
+  AbstractIntegrationDetailedView['props'],
+  AbstractIntegrationDetailedView['state']
+> {
+  tabs: Tab[] = ['overview'];
+
+  get integrationType() {
+    return 'document' as const;
+  }
+
+  get integration(): DocumentIntegration {
+    const {integrationSlug} = this.props.params;
+
+    return documentIntegrations[integrationSlug] || {};
+  }
+
+  get description() {
+    return this.integration.description || '';
```

Comment:
@NisanthanNanthakumar No need to default to an empty string if these are all required fields

Reply:
@scefali I think it might be good to keep it because the content is manually updated, and humans make mistakes.

Reply:
@NisanthanNanthakumar understood, I left a comment above on how proper TS typings prevents these kinds of human errors :)